### ISSUE TITLE
Bad class names order on server side rendering

### ISSUE THREAD

**Issue by bertho-zero** [JorgeBucaran/classwrap#10]
The order of my classes is different server side and client side, so I am forced to use [classnames](https://www.npmjs.com/package/classnames) in the meantime.

**Comment by JorgeBucaran** [comment]
@bertho-zero How can I reproduce this?

**Comment by JorgeBucaran** [comment]
Also please post the code or similar code that you were using. I need to see how you were invoking `classwrap` and what arguments you passed to it.

**Comment by bertho-zero** [comment]
I had to use something like:

```jsx
// kind === 'success' in my case

<button
  className={cw([ 'btn', 'btn-block', {
    [ `btn-${kind}` ]: !!kind
  }])}
>
  ...
</button>
```

On server I have `btn btn-success btn-block` and on client I have `btn btn-block btn-success`, or the other way round, I no longer have the example at hand.

**Comment by JorgeBucaran** [comment]
@bertho-zero I tested the result on the server and on the client on CodePen and got the same result in both instances. This is my code.

```js
console.log(
  cw([
    "btn",
    "btn-block",
    {
      [`btn-${kind}`]: !!kind
    }
  ])
) // => btn btn-block btn-success
```

Can you still reproduce the issue? And what Node.js version are / were you using?

**Comment by bertho-zero** [comment]
No I did not succeed in reproducing it, certainly a change that was made only on one side.

**Comment by JorgeBucaran** [comment]
@bertho-zero Thanks! FWIW I [added a test](https://github.com/JorgeBucaran/classwrap/blob/1522c02107401268b1a4a65daf52019f8975c223/tests/index.test.js#L24) for this and published 1.2.3 🎉
